### ISSUE TITLE
Pinned Laws menu item is spaced wrong

### ISSUE THREAD

**Issue by waldoj** [statedecoded/statedecoded#713]
![pinned](https://cloud.githubusercontent.com/assets/656758/25309584/23dc17ca-279f-11e7-8486-3fc3f8fd7f19.png)

Every LI in the menu is 89 pixels wide, with no margin or padding, which worked out fine for the existing menu items, but not for "Pinned Laws," in the above view. Apply some left padding to space them out equally, taking care to only do so in this particular responsive view.

**Comment by krues8dr** [comment]
What's your geometry here?  I'm not seeing this specific issue on any of the stock Chrome-emulation devices – it's correctly bumping to the vertical list by the time it gets this tiny.

**Comment by krues8dr** [comment]
Ah wait, it's not smaller devices, it's only wide ones.  And it gets worse – this layout was never designed to handle a fourth icon.  The easiest solution here is to push the menu breakpoint even further up, so there's plenty of room.

![screen shot 2017-04-23 at 11 29 44 am](https://cloud.githubusercontent.com/assets/127904/25314776/46761faa-2818-11e7-82c1-f5b819a06e28.jpg)

(In addition to making the `li`s wider.)

**Comment by krues8dr** [comment]
(This is a job for [Flexbox](http://caniuse.com/#feat=flexbox) but I am so not redoing the whole menu right before 1.0 drops.)

**Comment by krues8dr** [comment]
I am *not* in love with this solution I just pushed.  It basically punts and pushes the tablet size up to 1280px, so smaller lap/desktops will get the tablet size instead.  The header code has become spaghetti over time, so it's really hard for me to even start to unravel all the places it touches, so doing this piecemeal was a no-go; I just changed the variable in `Keanu` (the breakpoint library we use) instead.  It'll do, but it's ugly.  We should redo this whole thing at some point.
